backend/src/service/Jimmi.service.ts

```diff
@@ -0,0 +1,17 @@
+import Jimmi from './Jimmi.class.ts';
+
+const instances: Record<string, Jimmi> = {};
+
+export const createJimmi = (): Jimmi => {
+  const jimmi = new Jimmi();
+  instances[jimmi.id] = jimmi;
+  return jimmi;
+};
+
+export const getAllJimmiIds = (): string[] => {
+  return Object.keys(instances);
+};
+
+export const getJimmiBy = (id: string): Jimmi | undefined => {
```

Comment:
tsdoc strings would be nice :thinking:

Reply:
Also, is there a specific reason to prefer arrow functions over regular ones? :)

Reply:
Solved in 54cb2b9